### PR TITLE
Hook up start a move button with create move endpoint

### PR DESCRIPTION
**Pull request by amitch23**
## Description

This PR adds the functionality for the start a move button to create a move of type "COMBO" (hardcoded for now) and transition to `/moves/<move_id>`with the returned move id.

## Verification Steps

* [x] All tests pass.
* [x] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely) have been satisfied.
* [x] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI.
* [ ] This works in IE.

## References

* [Pivotal story](tbd) for this change

### CODE REVIEW

File: src/scenes/Landing/index.jsx [hunk 1]
```diff
@@ -21,24 +26,27 @@ export class Landing extends Component {
...
+          <button onClick={this.startMove}>Start a move</button>
         )}
       </div>
     );
   }
 }
 
-function getLegaleseRoute() {
+function getLegaleseRoute(currentMove) {
   //this is a horrible hack until we can get move id from server
```
Comment (tinyels) [inline src/scenes/Landing/index.jsx]: This comment can go.

File: src/scenes/Landing/index.jsx [hunk 2]
```diff
@@ -21,24 +26,27 @@ export class Landing extends Component {
...
-  const moveId = process.env.REACT_APP_MOVE_UUID;
-  return `moves/${moveId}/legalese`;
+  var moveID = currentMove.id;
+  return `moves/${moveID}/legalese`;
```
Comment (tinyels) [inline src/scenes/Landing/index.jsx]: Now that the basic workflow is in place, let's rename this method (or inline it) and change the route to `moves/${moveID}`

File: src/scenes/Landing/index.jsx (hunk 1 above)
Comment (amitch23) [reply src/scenes/Landing/index.jsx]: done

File: src/scenes/Landing/index.jsx (hunk 2 above)
Comment (amitch23) [reply src/scenes/Landing/index.jsx]: put it inline.

**Pull request merged**